src/components/TableDialog.vue

```diff
@@ -45,15 +45,16 @@
                 <v-layout wrap>
                   <v-flex>
                     <v-file-input
-                      :error-messages="fileUploadError"
+                      v-model="selectedFile"
```

Comment:
Could you explain what the changes in this commit accomplish? It's hard to tell from reading the code. In particular, it looks like the error message bit was stripped out and I'm not sure if it's been shifted to somewhere else. But overall I'd like to understand better what this does.

Reply:
Yes, it appeared to me that we were rolling our own validation and logic to handle file names, when that could be more easily handled by the vuetify component. 

The main goal of the commit was to update the table name when files were changed/removed so that you didn't upload a csv called "people.csv" to a table called "connections" (which I have done 🙃).

When I started making that change, I saw we should be using the `accept` and `v-model` attributes of the vuetify components,. which led to removing a ton of custom logic. For example, the table uploader should only allow .csv, then we don't need a custom function to check if the file is a csv; it's guaranteed that it will be a csv by the `accept` property.

Reply:
Great, thanks Jack!